user_docs/en/changes.md

```diff
@@ -45,6 +45,7 @@ Add-ons will need to be re-tested and have their manifest updated.
   * `BrowseModeTreeInterceptor` object has a new `getLinkTypeInDocument` method which accepts an URL to check the link type of the object
   * A `toggleBooleanValue` helper function has been added to `globalCommands`.
   It can be used in scripts to report the result when a boolean is toggled in `config.conf`
+* Removed the requirement to indent function parameter lists by two tabs from NVDA's Coding Standards, to be compatible with modern automatic linting. (#17126, XLTechie)
```

Comment:
I'm not sure a change log entry is needed for internal style changes to NVDA source code

Reply:
I wasn't going to do one at first. But then I thought that most of us read the coding standards quite infrequently. How else should devs who don't follow every PR, notice that there has been a change?
Especially since there has been a low key push from @josephsl and others, for add-on devs to try to keep their coding style synchronized with NVDA's, to make future adoption of add-ons into core easier.

I'm not going to be bothered if you drop it, but it literally is a "Change for developers".

Reply:
Fair enough - changes for devs is often add-on dev centric, which is why I was thinking this wasn't particularly helpful. The style is also enforced on devs automatically so the change really happened when we went to Ruff. Any thoughts @SaschaCowley ?

Reply:
I don't think one is strictly needed, but I don't think it's particularly out of place either. I think leaving it should be fine.